src/markdown-render.tsx

```diff
@@ -3,11 +3,14 @@ import { Source } from "@nteract/presentational-components";
 import React from "react";
 import ReactMarkdown from "react-markdown";
 
+import htmlParser = require("react-markdown/plugins/html-parser");
+
 import RemarkMathPlugin from "./remark-math";
 import AttachmentTransformer, {
   Attachments,
 } from "./attachment/attachment-transformer";
 
+const parseHTML = htmlParser();
```

Comment:
We can remove this now if we assume that it is being passed as part of `astPlugins`.

Reply:
Will do!